Assets/_BForBoss/_Core/Scripts/Managers/WeaponSceneManager.cs

```diff
@@ -12,15 +12,13 @@ public class WeaponSceneManager : MonoBehaviour
         private WeaponAnimationController _weaponAnimationController = null;
 
         [SerializeField] private EquipmentBehaviour _equipmentBehaviour = null;
-        [SerializeField] private AmmunitionCountViewBehaviour _ammunitionCountView = null;
```

Comment:
can you delete the script `AmmunitionCountViewBehaviour`

Reply:
Updated